internal/wire/datagram_frame.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/quic-go/quic-go/quicvarint"
 )
 
+// MaxDatagramSize is the maximum size of a DATAGRAM frame (RFC 9221).
+// By setting it to a large value, we allow all datagrams that fit into a QUIC packet.
+// The value is chosen such that it can still be encoded as a 2 byte varint.
+var MaxDatagramSize uint64 = 16383
```

Comment:
Have you considered making this a `protocol.ByteCount`?

Reply:
Sure, I will change that.